src/WebJobs.Script.WebHost/Extensions/FunctionMetadataExtensions.cs

```diff
@@ -170,8 +170,8 @@ internal static string GetScmBaseUrl()
         {
             // VFS APIs aren't exposed by the v1 runtime, so we must redirect
             // to the SCM site
-            string hostName = Environment.GetEnvironmentVariable(EnvironmentSettingNames.AzureWebsiteHostName);
-            
+            string hostName = HostNameProvider.Value;
```

Comment:
Since there isn't DI in v1 I chose to make this a static provider, to minimize code churn